apps/android/app/src/main/java/chat/simplex/app/views/chat/SendMsgView.kt

```diff
@@ -312,6 +184,157 @@ private fun NativeKeyboard(
   }
 }
 
+@Composable
+fun RecordVoiceView(
+  recState: MutableState<RecordingState>,
+  onAudioAdded: (filePath: String, durationMs: Int, finished: Boolean) -> Unit,
```

Comment:
the reason you need this callback is exactly the lack of finished state that would have these parameters